src/config_tools/config_tools.cc

```diff
@@ -295,4 +328,63 @@ void ValidateIOsConfig(TritonJson::Value &ios, const std::vector<IOConfig> &io_c
     ValidateIOConfig(io_object, io_config);
   }
 }
+
+
+void ValidateInputs(TritonJson::Value &ins, const std::vector<IOConfig> &in_configs) {
+  TRITON_CALL(ins.AssertType(common::TritonJson::ValueType::ARRAY));
+  ValidateAgainstTooManyInputs(ins, in_configs);
```

Comment:
We do. In `ValidateInputs` function there's 
```c++
auto ind = FindObjectByName(ins, in_config.name, &in_object);
if (!ind) {
  throw TritonError::InvalidArg(
    make_string("Missing config for \"", in_config.name, "\" input."));
}
```
It will be triggered if configuration specifies too few inputs.